CHANGELOG

```diff
@@ -1,6 +1,9 @@
-Version 5.0.0-alpha.1 (2022-11-30)
+Version 5.0.0-beta.1 (2022-02-02)
```

Comment:
On the Swift side, I kept all the pre-releases as separate items in the changelog, but I think this approach sounds good too, we can overwrite them and replace with the stable release when we get to it. Just adding a note here to sync that this is the way we will go about it, don't have any problem with it.

Reply:
Actually the V5.0.0-alpha.1 section had got in twice by accident, during the v4.1.1 release